### PR TITLE
[OPS-7462] Expose filename and uri as combined search filter, to allow searches on both.

### PR DESCRIPTION
**Pull request by cafuego**

### CODE REVIEW

File: config/views.view.files.yml [hunk 1]
```diff
@@ -189,8 +189,8 @@ display:
           label: Name
           exclude: false
           alter:
-            alter_text: false
-            text: ''
+            alter_text: true
+            text: "{{ filename }}<br />{{ uri }}"
```
Comment (lazysoundsystem) [inline config/views.view.files.yml:193]: I think `uri` needs to be added as a hidden field before this:
image

(I deployed security updates over what was on staging so I lost that as a reference.)

Comment (cafuego) [reply config/views.view.files.yml:193]: It *is* hidden as a field before that :-)

Comment (lazysoundsystem) [reply config/views.view.files.yml:193]: That's odd - on my local, I didn't see it as part of the view or the output, but when I added the field - as in the screenshot - it gave the output I was expecting.

Not sure what I missed but happy to approve this PR.

**Review by lazysoundsystem — APPROVED** [review]
Approved.

**Comment by lazysoundsystem** [comment]
Merged this to test it on staging.

**Comment by lazysoundsystem** [comment]
@cafuego  It's now on staging but I don't see the hidden `uri` field. Am I looking in the wrong place?

image

image

**Pull request merged**